src/main/java/de/hysky/skyblocker/skyblock/chocolatefactory/ChocolateFactorySolver.java

```diff
@@ -30,6 +30,7 @@ public class ChocolateFactorySolver extends ContainerSolver {
 	private static final Pattern CPS_PATTERN = Pattern.compile("([\\d,.]+) Chocolate per second");
 	private static final Pattern CPS_INCREASE_PATTERN = Pattern.compile("\\+([\\d,]+) Chocolate per second");
 	private static final Pattern COST_PATTERN = Pattern.compile("Cost ([\\d,]+) Chocolate");
+	private static final Pattern LEVEL_PATTERN = Pattern.compile("\\[(\\d+)]");
```

Comment:
This doesn't work for the coach jackrabbit, as it uses a different format for levels

Reply:
I'm guessing you don't have it unlocked or you'd probably have noticed - here's the lore of the item so you know what you're dealing with
![image](https://github.com/SkyblockerMod/Skyblocker/assets/81419447/0b067f92-a529-4e69-9b15-e71e18c517d9)